tools/coqdep/common.ml

```diff
@@ -248,8 +248,10 @@ let parse_META package f =
     close_in ic;
     Some (f, m)
   with
-  | Stream.Error msg -> Error.cannot_parse_meta_file package msg
   | Sys_error _msg -> None
+  (* findlib >= 1.9.3 uses its own Error exception, so we can't catch
+     it without bumping our version requirements *)
+  | _ -> Error.cannot_parse_meta_file package
```

Comment:
Can you also print the exception at least?

Reply:
IIRC the parse error from findlib is not very different  than "error" ;-) maybe it has a line number...

Reply:
Right, but this is also catching all exceptions so it could be something else.